src/validators/createMaxCharacterLength.js

```diff
@@ -0,0 +1,13 @@
+import i18n from '@dhis2/d2-i18n'
+import { createCharacterLengthRange } from './createCharacterLengthRange'
+
+const createMaxCharacterLength = upperBound =>
+    createCharacterLengthRange(
+        -Infinity,
```

Comment:
Why this and not 0?

Reply:
🤦‍♂ oh dear.... good find. I made this one and the createMaxNumber at the same time. Obviously a string cannot have a negative length.... Fixed in https://github.com/dhis2/ui-forms/pull/164/commits/0c6a693b55e243b30e556fcde36a1127b4b5ef44